example-deployment/docker/minimal.dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ros:foxy-ros-base-focal
+
+RUN apt-get update && apt-get install -y curl && \
+  curl -sL https://deb.nodesource.com/setup_12.x | bash - && \
+  apt-get update && apt-get install -y nodejs python3-pip
+
+RUN pip3 install pipenv
+
+COPY . /root/rmf-web
+SHELL ["bash", "-c"]
+
+RUN npm config set unsafe-perm && cd /root/rmf-web && \
+  CI=1 npm install -g lerna@4 && lerna bootstrap --scope=minimal
+
+RUN cd /root/rmf-web/packages/minimal && \
+  PUBLIC_URL='/minimal' \
+  REACT_APP_REPORTING_SERVER='https://example.com/logserver/api/v1' \
+  REACT_APP_AUTH_PROVIDER='keycloak' \
```

Comment:
i suspect you might need to add a few environment variables here, based on the contents of [this file](packages/minimal/.env):

```
  SKIP_PREFLIGHT_CHECK=true \
  REACT_APP_TRAJECTORY_SERVER=ws://example.com:8006 \
  REACT_APP_RMF_SERVER=https://example.com/rmf/api/v1 \
  REACT_APP_USER=admin \
```